upgrade/scripts/upgrade/util/update-customizations.sh

```diff
@@ -136,6 +136,9 @@ fi
 # Kube-prometheus-stack
 sed -i 's/prometheus-operator/kube-prometheus-stack/g' "$c"
 
+# Gitea
+yq d -i "$c" 'spec.kubernetes.services.gitea.cray-service.sqlCluster'
```

Comment:
In the effort of moving to `yq` version 4 at some point, I'd encourage you to use the newer binary and syntax.

```suggestion
yq4 -i 'del(.spec.kubernetes.services.gitea.cray-service.sqlCluster)' "$c"
```